pkg/cnab/config-adapter/testdata/porter-with-maintainers.yaml

```diff
@@ -25,8 +25,9 @@ credentials:
       - uninstall
 
 dependencies:
-  - name: mysql
-    reference: "getporter/azure-mysql:5.7"
+  requires:
```

Comment:
This change is unrelated to the PR. Without it, the "TestManifestConverter_generatedMaintainers" was failing.